lib/matplotlib/pyplot.py

```diff
@@ -244,7 +244,7 @@ def switch_backend(newbackend):
 
 def show(*args, **kw):
     """
-    Display a figure.
+    Display figures.
```

Comment:
```suggestion
    Display all figures.
```
Just to be sure that users don't get the idea these magic `*args, **kwargs` might be used to specify which figures.